src/index.html

```diff
@@ -2,13 +2,60 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <img src="./images/logo.png" alt="Moyo logo" class="logo">
+      <nav class="navigation">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a href="#" class="nav_link is-active">
+              apple
+              <div class="line"></div>
+            </a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">
+              samsung
+            </a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">
+              смартфоны и телефоны
+            </a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link" data-qa="hover">
+              ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">
+              гаджеты
+            </a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">
+              планшеты
+            </a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">
+              фото
+            </a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">
+              видео
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
   </body>
-</html>
+</html>
```

Comment:
```suggestion
</html>

```
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline#:~:text=If%20content%20is%20added%20to,you%20actually%20wanted%20to%20see.